jboss-kie-kieserver-webapp-files/added/WEB-INF/ejb-jar.xml

```diff
@@ -0,0 +1,96 @@
+<ejb-jar id="ejb-jar_ID" version="3.1"
+         xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee
+                          http://java.sun.com/xml/ns/javaee/ejb-jar_3_1.xsd">
+
+
+    <enterprise-beans>
+        <!-- we need to filter the JMS properties to override what is in KieServerMDB -->
+        <message-driven>
+            <ejb-name>KieServerMDB</ejb-name>
+            <ejb-class>org.kie.server.jms.KieServerMDB</ejb-class>
+            <transaction-type>Bean</transaction-type>
+            <activation-config>
+                <activation-config-property>
+                    <activation-config-property-name>destinationJndiName</activation-config-property-name>
+                    <activation-config-property-value>##QUEUE_REQUEST##</activation-config-property-value>
```

Comment:
Same as above regarding just using and replacing the original xml files with the proper values, searching for the values that are already in the file instead of a "##" vars.

Reply:
see my comment above.